boards/NUCLEOL476RG.py

```diff
@@ -50,7 +50,7 @@
   'usart' : 3,
   'spi' : 4,
   'i2c' : 3,
-  'adc' : 1,
+  'adc' : 2,
```

Comment:
According to this - there are 3 adc:
https://developer.mbed.org/platforms/ST-Nucleo-L476RG/

however unsure what the knock on effect of changing to 3 would be, so changed to 2 so the compile will work!